dev/build_release.py

```diff
@@ -63,7 +63,7 @@
 from spinnaker.run import run_quick
 
 
-SUBSYSTEM_LIST = ['clouddriver', 'orca', 'front50',
+SUBSYSTEM_LIST = ['clouddriver', 'orca', 'front50', 'halyard',
```

Comment:
Please add halyard to https://github.com/spinnaker/spinnaker/blob/master/dev/refresh_source.py also. I'd probably add this to `ADDITIONAL_SUBSYSTEMS` since it's not a microservice.